irohad/model/queries/get_transactions.hpp

```diff
@@ -38,16 +39,7 @@ namespace iroha {
        */
       std::string asset_id{};
     };
-
-    /**
-      * Query for getting transactions of account
-      */
-    struct GetAccountTransactions : Query {
-      /**
-       * Account identifier
-       */
-      std::string account_id{};
-    };
+    //TODO 06/11/17 motxx Add GetTransactions API
```

Comment:
Why u remove `GetAccountTransactions` query?

Reply:
This file will be changed as `GetTransactions` API.
`GetAccountTransactions` API will be at `get_account_transactions.hpp`
`GetAccountAssetTransactions` API will be at `get_account_asset_transactions.hpp`